sys/shell/cmds/gnrc_icmpv6_echo.c

```diff
@@ -207,7 +208,13 @@ static int _configure(int argc, char **argv, _ping_data_t *data)
                     /* intentionally falls through */
                 case 's':
                     if ((++i) < argc) {
-                        data->datalen = atoi(argv[i]);
+                        value = atoi(argv[i]);
+
+                        if(value < 0 || value > 65527) {
+                            printf("ICMPv6 datagram size should be in range <0, 65527>.\n");
```

Comment:
Nitpick: I'd prefer the following notation
```suggestion
                            printf("ICMPv6 datagram size should be in range 0-65527.\n");
```